napari/_vispy/layers/labels.py

```diff
@@ -291,6 +295,9 @@ def _on_colormap_change(self, event=None):
         else:
             self.node.cmap = VispyColormap(*colormap)
 
+    def _on_iso_gradient_mode_change(self):
+        self.node.iso_gradient_mode = self.layer.iso_gradient_mode
```

Comment:
`node` vs `_volume_node`: different choices in different methods, are they on purpose?

Reply:
Ah, I think I have a mistake here. These are in separate classes. I _thought_ `self.node` here was a `LabelLayerNode` class (defined below), which holds the two nodes (`_image_node` and `_volume_node`). 

I added the property/logic to that class so we wouldn't have to do an `isinstance` check here or access a private member like `self.node._volume_node`.

But in fact, I think `self.node` is actually set to one or the other `_image_node` or `_volume_node`, so I think we do need to add an `isinstance` check (as in `_on_rendering_change`).

Reply:
LGTM!